inc/functions_upload.php

```diff
@@ -675,12 +675,13 @@ function delete_uploaded_file($path = '')
 	$path = ltrim($path, '/');
 	$cdn_path = realpath($cdn_base_path . '/' . $path);
 
-
 	if($mybb->settings['usecdn'] && !empty($cdn_base_path))
 	{
 		$deleted = $deleted && @unlink($cdn_path);
```

Comment:
@Sama34 And where does it say that? http://docs.mybb.com/1.8/development/standards/

THe alternative is much more long-winded. This is just simple boolean evaluation...